app/src/main/res/layout-land/activity_login.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We can remove this file as the UI is not much different from the portrait one. Also starters let's make the app portrait only because adding support for landscape mode introduces bigger problems like state management so let's postpone it until we finish all the features